frontend/src/components/common/header/header.tsx

```diff
@@ -21,6 +27,10 @@ const Header: FC = () => {
 
   const handleVisibleChange = (): void => setVisible(!isVisible);
   const handleLogout = (): void => {
+    dispatch(eamActions.resetState());
+    dispatch(bsActions.resetState());
+    dispatch(scActions.resetState());
+    dispatch(slcActions.resetState());
```

Comment:
```suggestion
```
in any reducer you can add case for `authActions.logOut` it is easier than having shares for all scoops